Library/Homebrew/cmd/update.sh

```diff
@@ -368,25 +368,27 @@ user account:
 EOS
   fi
 
-  # we may want to use a Homebrew curl
+  set -e
```

Comment:
Sorry to contradict myself but I'm wondering if it wouldn't be better to remove `set -e` and add a comparable `odie "Curl must be installed and in your PATH!"` message.